staging/src/k8s.io/client-go/testing/fixture_test.go

```diff
@@ -233,3 +235,17 @@ func TestWatchAddAfterStop(t *testing.T) {
 		t.Errorf("test resource creation failed: %v", err)
 	}
 }
+
+func TestPatchWithMissingObject(t *testing.T) {
+	nodesResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "nodes"}
+
+	scheme := runtime.NewScheme()
+	codecs := serializer.NewCodecFactory(scheme)
+	o := NewObjectTracker(scheme, codecs.UniversalDecoder())
+	reaction := ObjectReaction(o)
+	action := NewRootPatchSubresourceAction(nodesResource, "node-1", types.StrategicMergePatchType, []byte(`{}`))
+	handled, node, err := reaction(action)
+	assert.True(t, handled)
+	assert.Nil(t, node)
+	assert.True(t, errors.IsNotFound(err), "error should be not found")
```

Comment:
"expected IsNotFound error" seems to be a better error message.

Reply:
Switched it to use `EqualError` :+1: